src/core_plugins/kibana/public/dashboard/top_nav/clone_modal.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  KuiModal,
+  KuiModalHeader,
+  KuiModalHeaderTitle,
+  KuiModalBody,
+  KuiModalBodyText,
+  KuiModalFooter,
+  KuiButton,
+  KuiModalOverlay
+} from 'ui_framework/components';
+
+export class DashboardCloneModal extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      newDashboardName: props.title
+    };
+  }
+
+  componentDidMount() {
+    this.confirmButton.focus();
```

Comment:
I haven't used it myself yet, but it looks like there is an `autoFocus` attribute that we can add to the button itself to make this work.

Tangentially, what made you choose to have the confirm button focused by default. From my naive perspective, we'd want the user to change the name of the Dashboard so there isn't a naming collision, so defaulting the focus to the textbox for them to make the changes would make more sense. This point might no longer be valid now that we're appending "Copy" to the name of the title with the most recent changes.